python/doc/examples/reliability_sensitivity/estimate_probability_directional_sampling.ipynb

```diff
@@ -11,35 +11,46 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this basic example we are going to estimate a failure probability with the directional simulation.\n",
+    "In this example we estimate a failure probability with the directional simulation algorithm provided by the `DirectionalSampling` class."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
     "\n",
-    "The directional simulation algorithm  operates in the standard space and is parametrized by two concepts:\n",
+    "The directional simulation algorithm  operates in the standard space based on:\n",
     "\n",
-    "1. a Root Strategy to evaluate the intersections of each direction with the limit state function and take into account the contribution of the direction to the event probability. The available corresponding classes are:\n",
-    "   - RiskyAndFast\n",
-    "   - MediumSafe\n",
-    "   - SafeAndSlow\n",
+    "1. a *root strategy* to evaluate the nearest failure point along each direction and take the contribution of each direction to the failure event probability into account. The available strategies are:\n",
+    "   - `RiskyAndFast`\n",
+    "   - `MediumSafe`\n",
+    "   - `SafeAndSlow`\n",
     " \n",
-    "2. a Sampling Strategy to chose directions in the standard space. The available corresponding classes are:\n",
-    "   - RandomDirection\n",
-    "   - OrthogonalDirection\n",
-    "\n",
+    "2. a *sampling strategy* to chose directions in the standard space. The available strategies are:\n",
```

Comment:
```suggestion
    "2. a *sampling strategy* to choose directions in the standard space. The available strategies are:\n",
```
Sorry I did not spot this typo the first time around.

Reply:
Damn! Fixed.